### PR TITLE
[MOS-339] Stop usbTick Timer

### PR DESCRIPTION
**Pull request by Maciej-Mudita**
After deninitization, the Timer was never stopped.

### CODE REVIEW

File: usb.cpp [hunk 1]
```diff
@@ -96,12 +96,14 @@ namespace bsp
...
         // Restart HW tick for resume operation
-        xTimerStart(usbTick, 1000);
+        xTimerStart(usbTick, 0);
         // Resume if suspended
         composite_resume(usbDeviceComposite);
 
         LOG_INFO("usbDeinit");
         composite_deinit(usbDeviceComposite);
+
+        xTimerStop(usbTick, 0);
```
Comment (mjan-dev) [inline usb.cpp:106]: shouldn't the time be stopped before deinit?

File: usb.cpp [hunk 2]
```diff
@@ -96,12 +96,14 @@ namespace bsp
     void usbDeinit()
     {
         // Restart HW tick for resume operation
-        xTimerStart(usbTick, 1000);
+        xTimerStart(usbTick, 0);
```
Comment (swichu91) [inline usb.cpp:99]: What is the point of such an operation? Do we want here to fire the timer immediately?

Comment (Maciej-Mudita) [reply usb.cpp:99]: The second argument is not the delay of the timer start, but xBlockTime, that is the time, in ticks, that the calling task should be held in the Blocked state to wait for the start command to be successfully sent to the timer command queue.

File: usb.cpp (hunk 1 above)
Comment (Maciej-Mudita) [reply usb.cpp:106]: i think it can be turned off before `composite_deinit` - tickTimer is needed for resume operation

File: usb.cpp (hunk 2 above)
Comment (swichu91) [reply usb.cpp:99]: Then, is it possible for `xTimerStart` request not reaching destination? Software timers are managed by the idle task which has its own requests queue.

Comment (Maciej-Mudita) [reply usb.cpp:99]: Unlikely, but yes, it is possible. FreeRTOS timers are handled in the `Tmr Svc` thread.

Comment (swichu91) [reply usb.cpp:99]: At least, handle return value :)

File: usb.cpp (hunk 1 above)
Comment (Maciej-Mudita) [reply usb.cpp]: @mjan-dev done

File: usb.cpp (hunk 2 above)
Comment (Maciej-Mudita) [reply usb.cpp]: @swichu91 done

File: usb.cpp [hunk 3]
```diff
@@ -26,17 +26,13 @@ extern "C"
...
-    #undef LOG_ERROR
-    #undef LOG_FATAL
     #undef LOG_CUSTOM
 
     #define LOG_PRINTF(...)
     #define LOG_TRACE(...)
     #define LOG_DEBUG(...)
     #define LOG_INFO(...)
     #define LOG_WARN(...)
-    #define LOG_ERROR(...)
```
Comment (Maciej-Mudita) [inline usb.cpp:38]: @mjan-dev @swichu91 I think errors should always be logged. What do you think about it?

Comment (swichu91) [reply usb.cpp:38]: Please check out the logging layer I recently added. You might find it useful.
`third-party/usb_stack/log.hpp`

Comment (Maciej-Mudita) [reply usb.cpp:38]: Ok, I changed it!

**Pull request merged**